extra/TestPlugin/build.gradle

```diff
@@ -6,35 +6,35 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 repositories {
-	jcenter()
-	
-	maven { url 'https://hub.spigotmc.org/nexus/content/repositories/public/' }
+    jcenter()
+
+    maven { url 'https://hub.spigotmc.org/nexus/content/repositories/public/' }
 }
 
 configurations.all {
-	resolutionStrategy.cacheChangingModulesFor 120, 'seconds'
+    resolutionStrategy.cacheChangingModulesFor 120, 'seconds'
 }
 
 dependencies {
-	implementation 'org.spigotmc:spigot-api:1.15.2-R0.1-SNAPSHOT'
+    implementation 'org.spigotmc:spigot-api:1.15.2-R0.1-SNAPSHOT'
 
-	testImplementation 'junit:junit:4.13.2'
+    testImplementation 'junit:junit:4.13.2'
 }
 
 jar.doLast() {
-	project.copy {
-		from 'build/libs/TestPlugin.jar'
-		into './'
-	}
+    project.copy {
+        from 'build/libs/TestPlugin.jar'
+        into './'
+    }
 }
 
 task deploy(dependsOn: 'jar') {
-	doLast {
-		project.copy() {
-			from 'build/libs/TestPlugin.jar'
-			into new File(gradle.deployDir, 'plugins/')
-		}
-	}
+    doLast {
+        project.copy() {
+            from 'build/libs/TestPlugin.jar'
+            into new File(gradle.deployDir, 'plugins/')
+        }
+    }
 }
```

Comment:
MockBukkit uses tabs, not spaces.

Reply:
Okay then we need to adapt the editorconfig

Reply:
![image](https://user-images.githubusercontent.com/24987877/180625581-2cf8f221-7838-4f11-8493-399b6cd3842c.png)

There's the culprit